OverlayPlugin.Core/NetworkProcessors/OverlayPluginLogLines.cs

```diff
@@ -165,32 +190,24 @@ private void LogException(string message)
                     return null;
                 }
 
-                if (opcodes.ContainsKey(version))
+                IOpcodeConfigEntry opcode = null;
+                opcode = GetOpcode(name, opcodesConfig, version, "config");
+                if (opcode == null)
                 {
-                    var versionOpcodes = opcodes[version];
-                    if (versionOpcodes.ContainsKey(name))
-                    {
-                        return versionOpcodes[name];
-                    }
-                    else
+                    opcode = GetOpcode(name, opcodesFile, version, "file");
+
+                    // Try once to get this remotely, but only if this opcode or version is missing.
+                    // TODO: we could consider getting this once always too, but for now
+                    // if we ever have an incorrect (but present) opcode, another release is required.
```

Comment:
Probably not applicable to this PR, but should this instead be a timestamp for last check and only check every X interval?

Considering how many people don't actually restart ACT very often.

Reply:
I think it could? I'm not too worried about it, though;  I feel like "stuff not working => try restarting ACT" is a pretty reasonable debugging method for most people.